### ISSUE TITLE
[cf-expandables] Lack of bottom spacing when using expandable-group and expandable__padding together

### ISSUE THREAD

**Issue by KimberlyMunoz** [cfpb/capital-framework#234]
> Some places that use the expandable-group and expandable__padding patterns together get an expandable with no bottom spacing.
> 
> See issue: https://github.com/cfpb/cfgov-refresh/issues/487

**Comment by jimmynotjim** [comment]
This is still an issue. @mistergone do you have time to tackle this before we release v4? If you add a link after the paragraph you can see exactly what's happening.

![screen shot 2017-01-12 at 2 49 41 pm](https://cloud.githubusercontent.com/assets/1280430/21907737/6f0c7e78-d8d6-11e6-9d70-fba9ce5983b9.png)

```
   <div class="o-expandable o-expandable__padded" data-bound="true">
        <button class="o-expandable_header o-expandable_target o-expandable_target__expanded" title="Expand content">
            <span class="o-expandable_header-left o-expandable_label">
                Expandable Header 3
            </span>
            <span class="o-expandable_header-right o-expandable_link">
                <span class="o-expandable_cue o-expandable_cue-open">
                    Show
                    <span class="cf-icon cf-icon-plus-round"></span>
                </span>
                <span class="o-expandable_cue o-expandable_cue-close">
                    Hide
                    <span class="cf-icon cf-icon-minus-round"></span>
                </span>
            </span>
        </button>
        <div class="o-expandable_content o-expandable_content__transition o-expandable_content__expanded" style="max-height: 120px;">
            <p>
                Lorem ipsum dolor sit amet, consectetur adipisicing
                elit. Neque ipsa voluptatibus soluta nobis unde quisquam
                temporibus magnam debitis quidem. Ducimus ratione
                corporis nesciunt earum vel est quaerat blanditiis
                dolore ipsa?
            </p>
            <a href="#">Link</a>
        </div>
    </div>
```

**Comment by jimmynotjim** [comment]
This was fixed in v4.2.1
